examples/datasource-streaming-websocket/streaming-websocket-plugin/provisioning/datasources/datasources.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: 1
```

Comment:
are you missing the URL in this provisioning file?

It is configured manually in the e2e tests but if someone were to use this provisioning to test the example the datasource won't work.